pkg/apis/crd/v1alpha1/types.go

```diff
@@ -439,6 +439,8 @@ type PacketCaptureSpec struct {
 	// for a capture session, and at least one `Pod` should be present either in the source or the destination.
 	Source      Source      `json:"source"`
 	Destination Destination `json:"destination"`
+	// Bidirection specifies whether to capture the return (response) traffic from the destination back to the source.
+	Bidirection bool `json:"bidirection"`
```

Comment:
maybe we could have an "enum" with such values: `SourceToDestination`, `DestinationToSource`, `Both` (and name the field `direction`).

What do you think?
cc @tnqn @hongliangl